package.json

```diff
@@ -47,7 +47,7 @@
     "coveralls": "^2.11.12",
     "cross-env": "^5.1.3",
     "css-loader": "^0.28.7",
-    "cypress": "^1.4.1",
+    "cypress": "^4.1.0",
```

Comment:
Can we upgrade cypress version to `^4.11.0`? More info here — https://github.com/JedWatson/react-select/pull/3957#issuecomment-667838702

Reply:
yep let me do that now :)